dataprofiler/data_readers/text_data.py

```diff
@@ -13,9 +14,14 @@
 class TextData(BaseData):
     """TextData class to save and load text files."""
 
-    data_type = "text"
+    data_type: Optional[str] = "text"
```

Comment:
same as above... in this file it would only be a `str` ?

Reply:
`BaseData` sets that field to `None`